tjmonopix2/system/board.py

```diff
@@ -9,8 +9,13 @@
 
 
 class DAQBoard(RegisterHardwareLayer):
-    ''' BDAQ readout board configuration
-    '''
+    """BDAQ readout board configuration
+
+    Parameters
+    ----------
+    RegisterHardwareLayer : class
+        Allows register read write operations
+    """
```

Comment:
This is effectively a basil driver for a custom firmware module that is configured using the basil bus system.